CMakeLists.txt

```diff
@@ -8,6 +8,13 @@ find_package(Kodi REQUIRED)
 
 set(ADP_SOURCES
 	src/main.cpp
+	src/codechandler/AVCCodecHandler.cpp
+	src/codechandler/CodecHandler.cpp
+	src/codechandler/HEVCCodecHandler.cpp
+	src/codechandler/MPEGCodecHandler.cpp
+	src/codechandler/TTMLCodecHandler.cpp
+	src/codechandler/VP9CodecHandler.cpp
+	src/codechandler/WebVTTCodecHandler.cpp
```

Comment:
all these should be moved in an appropriate CMakeLists file in codechandler folder

Reply:
Why?
My cmake is very basic but I think that then requires creating a new project, adding as dependency in main cmakelists... 
If the goal is to make it look less 'ugly' then we can do as other add-ons do and make an effective wildcard of all *.cpp/*.h files.

Reply:
I was thinking if with the future refactor's we could have many new files and this list starts to grow
but no problem if appropriate could be considered in the future

Reply:
Sure thing. For next refactor PR I'll change to the wildcard way, should then just be adding the folders.